Emby.Server.Implementations/Session/SessionWebSocketListener.cs

```diff
@@ -99,5 +130,130 @@ private void EnsureController(SessionInfo session, IWebSocketConnection connecti
             var controller = (WebSocketController)controllerInfo.Item1;
             controller.AddWebSocket(connection);
         }
+
+        /// <summary>
+        /// Called when a WebSocket is closed.
+        /// </summary>
+        /// <param name="sender">The WebSocket.</param>
+        /// <param name="e">The event arguments.</param>
+        private void _webSocket_Closed(object sender, EventArgs e)
```

Comment:
I see, I was following the style of `_serverManager_WebSocketConnected` that's a few lines above, but I like it better this way (`OnWebSocketClosed`).

Does `_serverManager_WebSocketConnected` follow another style because of `_serverManager` being a singleton? Or should all event handlers follow `OnEvent`? If the latter then I'll update the others as well (those added by the PR).

Reply:
Nevermind, I have now seen the proposed changes bellow for `_serverManager_*` as well.

Reply:
I don't see you renaming `_serverManager_WebSocketConnected`. Not that it was required, but would be nice to use consistent style :)

Reply:
Oh ok, I've renamed those in `SyncaplayManager`, renaming `_serverManager_WebSocketConnected` didn't even cross my mind as I thought it isn't related to the PR.